<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Test czcionki i koloru</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="mywindow.qrc" >:/images/qt.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTextEdit" name="textEdit" />
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_1" >
     <property name="text" >
      <string>Wybierz kolor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2" >
     <property name="text" >
      <string>Wybierz czcionkÄ™</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mywindow.qrc" />
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX" >
   <number>10</number>
  </property>
  <property name="gridDeltaY" >
   <number>10</number>
  </property>
  <property name="gridSnapX" >
   <bool>true</bool>
  </property>
  <property name="gridSnapY" >
   <bool>true</bool>
  </property>
  <property name="gridVisible" >
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
